CMakeLists.txt

```diff
@@ -182,8 +189,10 @@ if (PUGIXML_BUILD_APPLE_FRAMEWORK)
   set_target_properties(${libs} PROPERTIES
     FRAMEWORK TRUE
     FRAMEWORK_VERSION ${PROJECT_VERSION}
-    XCODE_ATTRIBUTE_PRODUCT_BUNDLE_IDENTIFIER com.zeux.pugixml
-    MACOSX_FRAMEWORK_IDENTIFIER com.zeux.pugixml
+    XCODE_ATTRIBUTE_PRODUCT_BUNDLE_IDENTIFIER "github.com/zeux/pugixml"
```

Comment:
I'm not sure how these get determined by why did this change?

Reply:
I updated the identifiers match the real URL (It'll be included in the `.plist` of the framework.
And `PUBLIC_HEADER` has to be specified. I overlooked it in the previous PR.